### ISSUE TITLE
Firefox plugin-container crash

### ISSUE THREAD

**Issue by NokiaGuy** [roblox-linux-wrapper/roblox-linux-wrapper#116]
```
0x10001aa1: int	$3
Modules:
Module	Address			Debug info	Name (105 modules)
PE	  340000-  35b000	Deferred        mozglue
PE	  360000-  395000	Deferred        sandboxbroker
PE	  3b0000-  3d2000	Deferred        robloxproxy
PE	  400000-  446000	Deferred        plugin-container
PE	  560000-  5e3000	Deferred        nprobloxproxy
PE	  800000- 2bc6000	Deferred        xul
PE	 2bd0000- 2d70000	Deferred        nss3
PE	10000000-10006000	Export          mozalloc
PE	4a800000-4a8cd000	Deferred        icuuc52
PE	4a900000-4aa0a000	Deferred        icuin52
PE	4ad00000-4b6ea000	Deferred        icudt52
ELF	7b800000-7ba69000	Deferred        kernel32<elf>
  \-PE	7b810000-7ba69000	\               kernel32
ELF	7bc00000-7bcf2000	Deferred        ntdll<elf>
  \-PE	7bc10000-7bcf2000	\               ntdll
ELF	7bf00000-7bf04000	Deferred        <wine-loader>
ELF	7e66a000-7e68f000	Deferred        imm32<elf>
  \-PE	7e670000-7e68f000	\               imm32
ELF	7e6d8000-7e701000	Deferred        libexpat.so.1
ELF	7e701000-7e73c000	Deferred        libfontconfig.so.1
ELF	7e73c000-7e764000	Deferred        libpng12.so.0
ELF	7e764000-7e77e000	Deferred        libz.so.1
ELF	7e77e000-7e81e000	Deferred        libfreetype.so.6
ELF	7e839000-7e8b5000	Deferred        advapi32<elf>
  \-PE	7e850000-7e8b5000	\               advapi32
ELF	7e8b5000-7e9cd000	Deferred        gdi32<elf>
  \-PE	7e8c0000-7e9cd000	\               gdi32
ELF	7e9cd000-7eb42000	Deferred        user32<elf>
  \-PE	7e9e0000-7eb42000	\               user32
ELF	7eb42000-7ebfb000	Deferred        winmm<elf>
  \-PE	7eb50000-7ebfb000	\               winmm
ELF	7ebfb000-7ec08000	Deferred        libnss_files.so.2
ELF	7ec08000-7ec14000	Deferred        libnss_nis.so.2
ELF	7ec14000-7ec2d000	Deferred        libnsl.so.1
ELF	7ef9f000-7efe5000	Deferred        libm.so.6
ELF	7efe6000-7f000000	Deferred        version<elf>
  \-PE	7eff0000-7f000000	\               version
ELF	f634e000-f6376000	Deferred        mpr<elf>
  \-PE	f6350000-f6376000	\               mpr
ELF	f6376000-f63f4000	Deferred        wininet<elf>
  \-PE	f6380000-f63f4000	\               wininet
ELF	f63f4000-f64ae000	Deferred        msvcr90<elf>
  \-PE	f6410000-f64ae000	\               msvcr90
ELF	f64ae000-f65f6000	Deferred        msvcp90<elf>
  \-PE	f64f0000-f65f6000	\               msvcp90
ELF	f663c000-f6642000	Deferred        libxfixes.so.3
ELF	f6642000-f664d000	Deferred        libxcursor.so.1
ELF	f664d000-f665d000	Deferred        libxi.so.6
ELF	f665d000-f6661000	Deferred        libxcomposite.so.1
ELF	f6661000-f666c000	Deferred        libxrandr.so.2
ELF	f666c000-f6677000	Deferred        libxrender.so.1
ELF	f6677000-f667d000	Deferred        libxxf86vm.so.1
ELF	f667d000-f669f000	Deferred        libxcb.so.1
ELF	f669f000-f67d3000	Deferred        libx11.so.6
ELF	f67d3000-f67e6000	Deferred        libxext.so.6
ELF	f6801000-f6896000	Deferred        winex11<elf>
  \-PE	f6810000-f6896000	\               winex11
ELF	f6896000-f68b5000	Deferred        pdh<elf>
  \-PE	f68a0000-f68b5000	\               pdh
ELF	f68b5000-f69fa000	Deferred        oleaut32<elf>
  \-PE	f68d0000-f69fa000	\               oleaut32
ELF	f69fa000-f6a12000	Deferred        wtsapi32<elf>
  \-PE	f6a00000-f6a12000	\               wtsapi32
ELF	f6a12000-f6ae2000	Deferred        crypt32<elf>
  \-PE	f6a20000-f6ae2000	\               crypt32
ELF	f6ae2000-f6b19000	Deferred        wintrust<elf>
  \-PE	f6af0000-f6b19000	\               wintrust
ELF	f6b19000-f6b8a000	Deferred        setupapi<elf>
  \-PE	f6b20000-f6b8a000	\               setupapi
ELF	f6b8a000-f6bc2000	Deferred        uxtheme<elf>
  \-PE	f6b90000-f6bc2000	\               uxtheme
ELF	f6bc2000-f6bd6000	Deferred        msimg32<elf>
  \-PE	f6bd0000-f6bd6000	\               msimg32
ELF	f6bd6000-f6c5a000	Deferred        rpcrt4<elf>
  \-PE	f6be0000-f6c5a000	\               rpcrt4
ELF	f6c5a000-f6d9c000	Deferred        ole32<elf>
  \-PE	f6c70000-f6d9c000	\               ole32
ELF	f6d9c000-f6de1000	Deferred        usp10<elf>
  \-PE	f6da0000-f6de1000	\               usp10
ELF	f6de1000-f6e5c000	Deferred        shlwapi<elf>
  \-PE	f6df0000-f6e5c000	\               shlwapi
ELF	f6e5c000-f70a8000	Deferred        shell32<elf>
  \-PE	f6e70000-f70a8000	\               shell32
ELF	f70a8000-f70d8000	Deferred        netapi32<elf>
  \-PE	f70b0000-f70d8000	\               netapi32
ELF	f70d8000-f70ff000	Deferred        iphlpapi<elf>
  \-PE	f70e0000-f70ff000	\               iphlpapi
ELF	f70ff000-f713a000	Deferred        ws2_32<elf>
  \-PE	f7110000-f713a000	\               ws2_32
ELF	f713a000-f7156000	Deferred        wsock32<elf>
  \-PE	f7140000-f7156000	\               wsock32
ELF	f7156000-f725f000	Deferred        msvcp120<elf>
  \-PE	f7190000-f725f000	\               msvcp120
ELF	f725f000-f7330000	Deferred        msvcr120<elf>
  \-PE	f7280000-f7330000	\               msvcr120
ELF	f7336000-f733b000	Deferred        libdl.so.2
ELF	f733b000-f74e9000	Deferred        libc.so.6
ELF	f74ea000-f7506000	Deferred        libpthread.so.0
ELF	f7507000-f7510000	Deferred        libnss_compat.so.2
ELF	f7511000-f7515000	Deferred        libxinerama.so.1
ELF	f7515000-f751c000	Deferred        libxdmcp.so.6
ELF	f751c000-f7520000	Deferred        libxau.so.6
ELF	f7521000-f76e7000	Dwarf           libwine.so.1
ELF	f76e9000-f770b000	Deferred        ld-linux.so.2
ELF	f770b000-f770c000	Deferred        [vdso].so
Threads:
process  tid      prio (all id:s are in hex)
0000000e services.exe
	00000023    0
	00000022    0
	0000001c    0
	00000014    0
	00000010    0
	0000000f    0
00000012 winedevice.exe
	0000001b    0
	00000018    0
	00000017    0
	00000013    0
00000019 winedevice.exe
	0000001f    0
	0000001e    0
	0000001a    0
00000020 plugplay.exe
	00000025    0
	00000024    0
	00000021    0
00000026 explorer.exe
	0000002a    0
	00000029    0
	00000028    0
	00000027    0
00000039 (D) C:\Program Files\Mozilla Firefox\plugin-container.exe
	0000005a    0 <==
	0000003d    0
0000004c crashreporter.exe
	0000004e    0
	0000004d    0
	00000016    0
System information:
    Wine build: wine-1.7.43 (Staging)
    Platform: i386
    Host system: Linux
    Host version: 3.13.0-53-generic
```

Ubuntu 14.04
Latest RLW, Wine, and Wine-Staging
Wine Preferences set to Wine-Staging by default
x86

**Comment by GLolol** [comment]
Duplicate of #105. Looking at the Gist there, it seems that the `copysignf` function was missing, which caused Firefox to crash. 

Relevant WineHQ bug: https://bugs.winehq.org/show_bug.cgi?id=38195 has been fixed in Wine's master (wine-mirror/wine@e1de0eeae416f31203ec6a666614a2b0f8ab33b1). Unfortunately, this hasn't been released yet, so you might have to wait a bit for a fix to be pushed to you.

**Comment by GLolol** [comment]
Closing as "fixed upstream, awaiting release" and not something we can really work around anyways.

**Comment by GLolol** [comment]
Update: this should be fixed in Wine 1.7.44, which was released earlier today. Again, you may have to wait for it to become available for your distro.

https://www.winehq.org/news/2015052901
